config/manager/patches/add_root_certificates.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocm-controller
+  labels:
+    control-plane: controller
+spec:
+  selector:
+    matchLabels:
+      app: ocm-controller
+  template:
+    spec:
+      containers:
+        - name: manager
+          image: open-component-model/ocm-controller
+          volumeMounts:
+            - mountPath: "/etc/ssl/certs"
+              name: "root-certificate"
+            - mountPath: "/certs"
+              name: "client-certificates"
+      volumes:
+        - name: "root-certificate"
+          secret:
+            secretName: "developer-root-certificate"
+            items:
+              - key: "ca-certificates.crt"
+                path: "ca-certificates.crt"
+        - name: "client-certificates"
+          secret:
+            secretName: "registry-certs"
+            items:
+              - key: "server.pem"
+                path: "server.pem"
+              - key: "server-key.pem"
+                path: "server-key.pem"
```

Comment:
Any reason to not put these all in the same secret? That would be how I usually set TLS secrets configured.

Reply:
Because, strictly, the root certificate is not needed in the registry container.

Reply:
I guess I can add it in one secret.